indexer-app/src/cmr/indexer/data/collection_granule_aggregation_cache.clj

```diff
@@ -219,16 +219,18 @@
          (c/set-value cache coll-gran-aggregate-cache-key
                       (coll-gran-aggregates->cachable-value merged-map))
 
-         (when (seq updated-collections)
-          (info "Reindexing collections found with updated temporal values since last ingest:"
-                (pr-str updated-collections))
+         (try
```

Comment:
I know this isn't your issue, but on line 219, we set the cache value regardless if the reindexing worked or not:
         (c/set-value cache coll-gran-aggregate-cache-key
                      (coll-gran-aggregates->cachable-value merged-map))

Shouldn't this only be done if no exceptions occurred?

Reply:
I'm going to pull this out and make a ticket for it so that we can look at ramifications